arches/app/templates/views/search.htm

```diff
@@ -33,7 +33,7 @@
                     <div class="col-xs-6">
                         <!-- Result Count and ordering Block -->
                         <div class="search-count-container">
-                            <p class="search-title" aria-live="polite" data-bind="text: {% quoted_trans 'Results: ' %} + total()"></p>
+                            <p class="search-title" aria-live="polite" data-bind="text: $root.translations.shownTotalSearchResults(sharedStateObject?.searchResults?.results?.hits?.hits?.length, total())"></p>
```

Comment:
@johnatawnclementawn Instead of referencing this nested object value here for the returned hits, I'd create an observable in the .js file and use that instead here.